docs/deployment/README.md

```diff
@@ -37,6 +37,14 @@ The scripts in this repository are meant to be used for development/testing purp
 [Security best practices for Amazon Cognito user pools](https://docs.aws.amazon.com/AmazonS3/latest/userguide/security-best-practices.html)  
 [Security in Amazon Elastic Load Balancing (ELB)](https://docs.aws.amazon.com/elasticloadbalancing/latest/userguide/security.html)
 
+## CloudWatch Logging and Container Insights
+Amazon EKS offers Container Insights using Amazon CloudWatch which monitors your Amazon Web Services (AWS) resources and the applications you run on AWS in real time. You can use CloudWatch to collect and track metrics, which are variables you can measure for your resources and applications. FluentBit can be used as the DaemonSet to send logs to CloudWatch Logs, and will send the following logs which contain metrics such as individual pod logs. Install AWS CloudWatch by following their [documentation](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/deploy-container-insights-EKS.html).
+   - /aws/containerinsights/Cluster_Name/application
+
+   - /aws/containerinsights/Cluster_Name/host
+
+   - /aws/containerinsights/Cluster_Name/dataplane
```

Comment:
These are the expected logs that the user will see in cloudwatch after installing fluentbit. Can be left out since they would see them in the documentation I linked as well if they read far enough to verify installation. 

Or are you asking what kind of metrics are in each of these bullet points?